src/astro/databases/google/bigquery.py

```diff
@@ -306,6 +307,43 @@ def get_project_id(self, target_table) -> str:
         except AttributeError:
             raise ValueError(f"conn_id {target_table.conn_id} has no project id")
 
+    def load_local_file_to_bigquery(
+        self,
+        source_file: File,
+        target_table: Table,
+        if_exists: LoadExistStrategy = "replace",
+        native_support_kwargs: Optional[Dict] = None,
+        **kwargs,
+    ) -> None:
+        """Transfer data from local to bigquery"""
+        native_support_kwargs = native_support_kwargs or {}
+        # We need to maintain file_type to biqquery_format and not use NATIVE_PATHS_SUPPORTED_FILE_TYPES
+        # because the load_table_from_file expects 'JSON' value for ndjson file.
+        file_types_to_bigquery_format = {
```

Comment:
@tatiana I think we should keep it in this place only. Since they are both used for the same thing, the google upstream API requires us to pass different values for NDJSON - NEWLINE_DELIMITED_JSON/JSON. So keeping mapping here will avoid some confusion.